Common/TableProducer/multiplicityTable.cxx

```diff
@@ -163,7 +180,7 @@ struct MultiplicityTableTaskIndexed {
       }
     }
 
-    if (fabs(collision.posZ()) < 15.0f) {
+    if (fabs(collision.posZ()) < 15.0f && lCalibLoaded) {
```

Comment:
Same comment than above, you wont be able to run without calibration
But again, it does not harm